edgelet/edgelet-http/src/util/incoming.rs

```diff
@@ -5,8 +5,7 @@ use std::io;
 use std::sync::Mutex;
 
 use futures::{Poll, Stream};
-#[cfg(unix)]
-use tokio::io::{Error as TokioIoError, ErrorKind as TokioIoErrorKind};
+use log::Level;
```

Comment:
As the CI failure says, this import and the `log_failure` one below need to be `#[cfg(unix)]`

Reply:
Yeah, I noticed that. Thanks!